File: reverse_word_in_string.java
import javax.sql.rowset.spi.SyncResolver;

class reverse_word_in_string {

  String reverse_word(String word) {
    String s2 = "";
    for (int i = word.length() - 1; i >= 0; i--) {
      s2 = s2 + word.charAt(i);
    }

    return s2;
  }

  public static void main(String[] args) {
    reverse_word_in_string ss = new reverse_word_in_string();
    String s = "Hello Harsh";

    String[] arr;
    arr = s.split(" ");

    String s2 = "";

    for (int i = 0; i < arr.length; i++) {
      System.out.println(ss.reverse_word(arr[i]));
    }
    for (int i = 0; i < arr.length; i++) {
      s2 = " " + ss.reverse_word(arr[i]) + s2;
    }

    System.out.println(s2);
  }
}

File: reverse_word_in_string.cpp
#include <iostream>
#include <vector>
#include <string>

using namespace std;
string rev(string s)
{
    string s2="";
    for (int i = s.size()-1; i >=0; i--)
    {
        s2+=s[i];
    }
    return s2;
    
}

int main()
{
    string s = "Hello Harsh";

    vector<string> words;

    string s2 = "";

    for (int i = 0; i < s.size(); i++)
    {
        if (s[i] == ' ')
        {
            words.push_back(rev(s2));
            s2 = "";
        }
        else
        {
            s2 += s[i];
        }
    }

    words.push_back(rev(s2));

    for (int i = 0; i < words.size(); i++)
    {
        cout << words[i] << " ";
    }

    return 0;
}
